### PR TITLE
Parallel

### PR DESCRIPTION
**Pull request by jburel**
* Add script to show how to analyze data in parallel using a local cluster.
* Add doc page.
* Adjust notebook to show progress when loading data

When running the script, you should create a conda env locally
i.e. ``conda env create --name ilastik -f binder/environment.yml``

### CODE REVIEW

File: scripts/pixel_classification_zarr_parallel.py [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+        (
+            "Raw Data",
+            [PreloadedArrayDatasetInfo(preloaded_array=input_data)],
+        )])
+    return shell.workflow.batchProcessingApplet.run_export(data, export_to_array=True)  # noqa
+
+
+# Prepare-call
+def prepare(client, images, ilastik_project):
+    futures = []
```
Comment (mtbc) [inline scripts/pixel_classification_zarr_parallel.py]: Could consider a list comprehension.

**Comment by pwalczysko** [comment]
> Adjust notebook to show progress when loading data

If repo2docker and mybinder do not work, than this is not testable ?

**Comment by jburel** [comment]
@pwalczysko 
* to test the notebook you can use repo2docker.
* to test the script you cannot.

**Comment by pwalczysko** [comment]
The script (locally) "worked". Adding a snippet out of the output (the output is very confusing, no idea what really happened, and what should have happened.

```
DEBUG lazyflow.operators.classifierOperators: Features took 0.309048 seconds. Prediction took 0.03204 seconds. Subregion: start '[184, 276, 276, 0]' stop '[239, 281, 325, 2]'
201.86128211021423
done
/Users/pwalczysko/miniconda3/envs/ilastik-test/lib/python3.7/multiprocessing/semaphore_tracker.py:144: UserWarning: semaphore_tracker: There appear to be 1 leaked semaphores to clean up at shutdown
  len(cache))
/Users/pwalczysko/miniconda3/envs/ilastik-test/lib/python3.7/multiprocessing/semaphore_tracker.py:144: UserWarning: semaphore_tracker: There appear to be 1 leaked semaphores to clean up at shutdown
  len(cache))
/Users/pwalczysko/miniconda3/envs/ilastik-test/lib/python3.7/multiprocessing/semaphore_tracker.py:144: UserWarning: semaphore_tracker: There appear to be 1 leaked semaphores to clean up at shutdown
  len(cache))
/Users/pwalczysko/miniconda3/envs/ilastik-test/lib/python3.7/multiprocessing/semaphore_tracker.py:144: UserWarning: semaphore_tracker: There appear to be 1 leaked semaphores to clean up at shutdown
  len(cache))
```

**Comment by pwalczysko** [comment]
When testing the notebook (the idr one)

repo2docker

I can see the progress bar, and it is a nice addition.

But, when I go through the whole notebook, then, after I start the analysis cell itself, I twice got


Screen Shot 2020-05-11 at 11 45 49

**Comment by jburel** [comment]
hummm. This is the only change in this notebook

**Comment by jburel** [comment]
I ran ``pixel_classification_zarr_idr.ipynb`` without problem

**Comment by jburel** [comment]
@pwalczysko I saved the first plane as png. We could save it back as an omero image, this will take some time so not a great ad but I think we could look into saving as a Zarr file later on. For the moment I save the first plane as png.

**Comment by pwalczysko** [comment]
> I ran pixel_classification_zarr_idr.ipynb without problem

I believe so. I am starting to gather experience with the "Kernel restart". Hard to understand, impossible to repeat on demand, very intermittent.

**Comment by pwalczysko** [comment]
>  saved the first plane as png. 

Yes, this could be a solution. And the script still works. But I have no idea where to look for that png now, the output (see below) does not help

<details>

```
(ilastik-test2) pwalczysko@ls31618~/Work/omero-guide-ilastik/scripts$ python pixel_classification_zarr_parallel.py 
Host [wss://outreach.openmicroscopy.org/omero-ws]: 
Username [trainer-1]: 
Password: 
Dataset ID [6161]: 
INFO ilastik_main: Starting ilastik from "/Users/pwalczysko/miniconda3/envs/ilastik-test2".
INFO ilastik_main: Starting ilastik from "/Users/pwalczysko/miniconda3/envs/ilastik-test2".
Starting ilastik from "/Users/pwalczysko/miniconda3/envs/ilastik-test2".
Starting ilastik from "/Users/pwalczysko/miniconda3/envs/ilastik-test2".
INFO ilastik_main: Resetting lazyflow thread pool with 2 threads.
INFO ilastik_main: Resetting lazyflow thread pool with 2 threads.
INFO ilastik_main: Configuring lazyflow RAM limit to 2.0GiB
INFO lazyflow.utility.memory: Available memory set to 2.0GiB
INFO ilastik_main: Configuring lazyflow RAM limit to 2.0GiB
INFO lazyflow.utility.memory: Available memory set to 2.0GiB
WARNING 2020-05-11 15:01:23,052 opConservationTracking 71395 123145564864512 Could not find any ILP solver
WARNING 2020-05-11 15:01:23,053 opConservationTracking 71396 123145526444032 Could not find any ILP solver
WARNING 2020-05-11 15:01:23,061 opStructuredTracking 71395 123145564864512 Could not find any ILP solver
WARNING 2020-05-11 15:01:23,063 structuredTrackingWorkflow 71395 123145564864512 Could not find any learning solver. Tracking will use flow-based solver (DPCT). Learning for tracking will be disabled!
WARNING 2020-05-11 15:01:23,063 opStructuredTracking 71396 123145526444032 Could not find any ILP solver
WARNING 2020-05-11 15:01:23,064 structuredTrackingWorkflow 71396 123145526444032 Could not find any learning solver. Tracking will use flow-based solver (DPCT). Learning for tracking will be disabled!
INFO ilastik.shell.projectManager: Opening Project: ../notebooks/pipelines/pixel-class-133.ilp
WARNING projectManager.py(158): H5pyDeprecationWarning: dataset.value has been deprecated. Use dataset[()] instead.
WARNING projectManager.py(168): H5pyDeprecationWarning: dataset.value has been deprecated. Use dataset[()] instead.
INFO ilastik.shell.projectManager: Opening Project: ../notebooks/pipelines/pixel-class-133.ilp
WARNING projectManager.py(158): H5pyDeprecationWarning: dataset.value has been deprecated. Use dataset[()] instead.
WARNING projectManager.py(168): H5pyDeprecationWarning: dataset.value has been deprecated. Use dataset[()] instead.
WARNING dataSelectionSerializer.py(385): H5pyDeprecationWarning: dataset.value has been deprecated. Use dataset[()] instead.
WARNING featureSelectionSerializer.py(91): H5pyDeprecationWarning: dataset.value has been deprecated. Use dataset[()] instead.
WARNING featureSelectionSerializer.py(96): H5pyDeprecationWarning: dataset.value has been deprecated. Use dataset[()] instead.
WARNING featureSelectionSerializer.py(102): H5pyDeprecationWarning: dataset.value has been deprecated. Use dataset[()] instead.
WARNING featureSelectionSerializer.py(122): H5pyDeprecationWarning: dataset.value has been deprecated. Use dataset[()] instead.
WARNING dataSelectionSerializer.py(385): H5pyDeprecationWarning: dataset.value has been deprecated. Use dataset[()] instead.
WARNING featureSelectionSerializer.py(91): H5pyDeprecationWarning: dataset.value has been deprecated. Use dataset[()] instead.
WARNING featureSelectionSerializer.py(96): H5pyDeprecationWarning: dataset.value has been deprecated. Use dataset[()] instead.
WARNING featureSelectionSerializer.py(102): H5pyDeprecationWarning: dataset.value has been deprecated. Use dataset[()] instead.
WARNING featureSelectionSerializer.py(122): H5pyDeprecationWarning: dataset.value has been deprecated. Use dataset[()] instead.
WARNING opSimpleBlockedArrayCache.py(43): FutureWarning: Conversion of the second argument of issubdtype from `dtype` to `np.generic` is deprecated. In future, it will be treated as `np.object_ == np.dtype(dtype).type`.
WARNING opSimpleBlockedArrayCache.py(43): FutureWarning: Conversion of the second argument of issubdtype from `dtype` to `np.generic` is deprecated. In future, it will be treated as `np.object_ == np.dtype(dtype).type`.
WARNING opSlicedBlockedArrayCache.py(141): FutureWarning: Conversion of the second argument of issubdtype from `dtype` to `np.generic` is deprecated. In future, it will be treated as `np.object_ == np.dtype(dtype).type`.
WARNING opSlicedBlockedArrayCache.py(141): FutureWarning: Conversion of the second argument of issubdtype from `dtype` to `np.generic` is deprecated. In future, it will be treated as `np.object_ == np.dtype(dtype).type`.
WARNING featureSelectionSerializer.py(169): H5pyDeprecationWarning: dataset.value has been deprecated. Use dataset[()] instead.
WARNING featureSelectionSerializer.py(169): H5pyDeprecationWarning: dataset.value has been deprecated. Use dataset[()] instead.
WARNING pixelClassificationSerializer.py(227): H5pyDeprecationWarning: dataset.value has been deprecated. Use dataset[()] instead.
WARNING pixelClassificationSerializer.py(227): H5pyDeprecationWarning: dataset.value has been deprecated. Use dataset[()] instead.
WARNING opSimpleBlockedArrayCache.py(43): FutureWarning: Conversion of the second argument of issubdtype from `dtype` to `np.generic` is deprecated. In future, it will be treated as `np.object_ == np.dtype(dtype).type`.
WARNING opSlicedBlockedArrayCache.py(141): FutureWarning: Conversion of the second argument of issubdtype from `dtype` to `np.generic` is deprecated. In future, it will be treated as `np.object_ == np.dtype(dtype).type`.
WARNING opSimpleBlockedArrayCache.py(43): FutureWarning: Conversion of the second argument of issubdtype from `dtype` to `np.generic` is deprecated. In future, it will be treated as `np.object_ == np.dtype(dtype).type`.
WARNING opSlicedBlockedArrayCache.py(141): FutureWarning: Conversion of the second argument of issubdtype from `dtype` to `np.generic` is deprecated. In future, it will be treated as `np.object_ == np.dtype(dtype).type`.
INFO ilastik.applets.batchProcessing.batchProcessingApplet: Exporting to in-memory array.
INFO lazyflow.utility.bigRequestStreamer: Estimated RAM usage per pixel is 504.0B * safety factor (2.0)
INFO lazyflow.utility.bigRequestStreamer: determining blockshape assuming available_ram is 1.5GiB, split between 2 threads
INFO lazyflow.utility.bigRequestStreamer: Chose blockshape: (1, 92, 92, 92, 2)
INFO lazyflow.utility.bigRequestStreamer: Estimated RAM usage per block is 748.6MiB
INFO ilastik.applets.batchProcessing.batchProcessingApplet: Exporting to in-memory array.
INFO lazyflow.utility.bigRequestStreamer: Estimated RAM usage per pixel is 504.0B * safety factor (2.0)
INFO lazyflow.utility.bigRequestStreamer: determining blockshape assuming available_ram is 1.5GiB, split between 2 threads
INFO lazyflow.utility.bigRequestStreamer: Chose blockshape: (1, 92, 92, 92, 2)
INFO lazyflow.utility.bigRequestStreamer: Estimated RAM usage per block is 748.6MiB
WARNING arraytypes.py(1263): FutureWarning: Using a non-tuple sequence for multidimensional indexing is deprecated; use `arr[tuple(seq)]` instead of `arr[seq]`. In the future this will be interpreted as an array index, `arr[np.array(seq)]`, which will result either in an error or a different result.
WARNING arraytypes.py(1269): FutureWarning: Using a non-tuple sequence for multidimensional indexing is deprecated; use `arr[tuple(seq)]` instead of `arr[seq]`. In the future this will be interpreted as an array index, `arr[np.array(seq)]`, which will result either in an error or a different result.
WARNING arraytypes.py(1263): FutureWarning: Using a non-tuple sequence for multidimensional indexing is deprecated; use `arr[tuple(seq)]` instead of `arr[seq]`. In the future this will be interpreted as an array index, `arr[np.array(seq)]`, which will result either in an error or a different result.
WARNING arraytypes.py(1269): FutureWarning: Using a non-tuple sequence for multidimensional indexing is deprecated; use `arr[tuple(seq)]` instead of `arr[seq]`. In the future this will be interpreted as an array index, `arr[np.array(seq)]`, which will result either in an error or a different result.
DEBUG lazyflow.operators.classifierOperators: Features took 4.143612 seconds. Prediction took 2.173685 seconds. Subregion: start '[0, 0, 0, 0]' stop '[92, 92, 92, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 4.068096 seconds. Prediction took 2.222282 seconds. Subregion: start '[0, 0, 0, 0]' stop '[92, 92, 92, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 7.1240310000000004 seconds. Prediction took 3.017904 seconds. Subregion: start '[0, 0, 92, 0]' stop '[92, 92, 184, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 7.121469 seconds. Prediction took 3.077437 seconds. Subregion: start '[0, 0, 92, 0]' stop '[92, 92, 184, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 4.962324 seconds. Prediction took 2.76301 seconds. Subregion: start '[0, 0, 184, 0]' stop '[92, 92, 271, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 5.311535 seconds. Prediction took 2.498324 seconds. Subregion: start '[0, 0, 184, 0]' stop '[92, 92, 276, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 4.461366 seconds. Prediction took 2.173757 seconds. Subregion: start '[0, 0, 276, 0]' stop '[92, 92, 325, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 5.787573 seconds. Prediction took 2.779575 seconds. Subregion: start '[0, 92, 0, 0]' stop '[92, 184, 92, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 5.091403 seconds. Prediction took 2.754603 seconds. Subregion: start '[0, 92, 0, 0]' stop '[92, 184, 92, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 6.730044 seconds. Prediction took 2.692716 seconds. Subregion: start '[0, 92, 92, 0]' stop '[92, 184, 184, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 6.972281 seconds. Prediction took 3.303576 seconds. Subregion: start '[0, 92, 92, 0]' stop '[92, 184, 184, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 6.168974 seconds. Prediction took 2.535702 seconds. Subregion: start '[0, 92, 184, 0]' stop '[92, 184, 271, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 5.801564 seconds. Prediction took 2.667424 seconds. Subregion: start '[0, 92, 184, 0]' stop '[92, 184, 276, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 6.00965 seconds. Prediction took 2.50106 seconds. Subregion: start '[0, 184, 0, 0]' stop '[92, 275, 92, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 4.803808 seconds. Prediction took 1.588311 seconds. Subregion: start '[0, 92, 276, 0]' stop '[92, 184, 325, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 5.801462 seconds. Prediction took 2.606074 seconds. Subregion: start '[0, 184, 92, 0]' stop '[92, 275, 184, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 4.728101 seconds. Prediction took 2.688635 seconds. Subregion: start '[0, 184, 0, 0]' stop '[92, 276, 92, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 6.026465 seconds. Prediction took 2.389759 seconds. Subregion: start '[0, 184, 184, 0]' stop '[92, 275, 271, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 6.013621 seconds. Prediction took 3.279094 seconds. Subregion: start '[0, 184, 92, 0]' stop '[92, 276, 184, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 6.270307 seconds. Prediction took 3.070531 seconds. Subregion: start '[92, 0, 0, 0]' stop '[184, 92, 92, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 5.364278 seconds. Prediction took 3.086495 seconds. Subregion: start '[0, 184, 184, 0]' stop '[92, 276, 276, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 1.170909 seconds. Prediction took 0.209027 seconds. Subregion: start '[0, 276, 0, 0]' stop '[92, 281, 92, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 4.213954 seconds. Prediction took 1.2748439999999999 seconds. Subregion: start '[0, 184, 276, 0]' stop '[92, 276, 325, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 0.68102 seconds. Prediction took 0.135133 seconds. Subregion: start '[0, 276, 92, 0]' stop '[92, 281, 184, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 0.862888 seconds. Prediction took 0.211892 seconds. Subregion: start '[0, 276, 184, 0]' stop '[92, 281, 276, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 6.137532 seconds. Prediction took 2.647671 seconds. Subregion: start '[92, 0, 92, 0]' stop '[184, 92, 184, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 0.593134 seconds. Prediction took 0.112281 seconds. Subregion: start '[0, 276, 276, 0]' stop '[92, 281, 325, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 5.979648 seconds. Prediction took 3.108258 seconds. Subregion: start '[92, 0, 184, 0]' stop '[184, 92, 271, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 4.238621 seconds. Prediction took 2.595573 seconds. Subregion: start '[92, 0, 0, 0]' stop '[184, 92, 92, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 6.869389 seconds. Prediction took 2.618112 seconds. Subregion: start '[92, 92, 0, 0]' stop '[184, 184, 92, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 7.311976 seconds. Prediction took 3.042657 seconds. Subregion: start '[92, 0, 92, 0]' stop '[184, 92, 184, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 6.938239 seconds. Prediction took 2.42358 seconds. Subregion: start '[92, 92, 92, 0]' stop '[184, 184, 184, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 5.681188 seconds. Prediction took 2.521359 seconds. Subregion: start '[92, 0, 184, 0]' stop '[184, 92, 276, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 4.788495 seconds. Prediction took 2.156958 seconds. Subregion: start '[92, 0, 276, 0]' stop '[184, 92, 325, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 6.297701 seconds. Prediction took 2.6860720000000002 seconds. Subregion: start '[92, 92, 184, 0]' stop '[184, 184, 271, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 6.390679 seconds. Prediction took 2.594143 seconds. Subregion: start '[92, 184, 0, 0]' stop '[184, 275, 92, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 5.463471 seconds. Prediction took 2.674155 seconds. Subregion: start '[92, 92, 0, 0]' stop '[184, 184, 92, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 6.401992 seconds. Prediction took 2.575221 seconds. Subregion: start '[92, 184, 92, 0]' stop '[184, 275, 184, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 7.240869 seconds. Prediction took 3.077116 seconds. Subregion: start '[92, 92, 92, 0]' stop '[184, 184, 184, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 5.849602 seconds. Prediction took 2.370666 seconds. Subregion: start '[92, 184, 184, 0]' stop '[184, 275, 271, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 6.646379 seconds. Prediction took 2.325695 seconds. Subregion: start '[92, 92, 184, 0]' stop '[184, 184, 276, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 4.067617 seconds. Prediction took 1.541865 seconds. Subregion: start '[184, 0, 0, 0]' stop '[236, 92, 92, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 5.038194 seconds. Prediction took 1.7988309999999998 seconds. Subregion: start '[92, 92, 276, 0]' stop '[184, 184, 325, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 2.948106 seconds. Prediction took 1.373847 seconds. Subregion: start '[184, 0, 92, 0]' stop '[236, 92, 184, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 3.186303 seconds. Prediction took 1.582867 seconds. Subregion: start '[184, 0, 184, 0]' stop '[236, 92, 271, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 4.869753 seconds. Prediction took 2.70531 seconds. Subregion: start '[92, 184, 0, 0]' stop '[184, 276, 92, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 3.623201 seconds. Prediction took 1.431309 seconds. Subregion: start '[184, 92, 0, 0]' stop '[236, 184, 92, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 3.8002700000000003 seconds. Prediction took 1.3883809999999999 seconds. Subregion: start '[184, 92, 92, 0]' stop '[236, 184, 184, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 6.448737 seconds. Prediction took 3.005325 seconds. Subregion: start '[92, 184, 92, 0]' stop '[184, 276, 184, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 3.396871 seconds. Prediction took 1.32729 seconds. Subregion: start '[184, 92, 184, 0]' stop '[236, 184, 271, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 3.218156 seconds. Prediction took 1.3671 seconds. Subregion: start '[184, 184, 0, 0]' stop '[236, 275, 92, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 5.534859 seconds. Prediction took 2.481503 seconds. Subregion: start '[92, 184, 184, 0]' stop '[184, 276, 276, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 1.418213 seconds. Prediction took 0.47696 seconds. Subregion: start '[92, 276, 0, 0]' stop '[184, 281, 92, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 4.435412 seconds. Prediction took 1.275803 seconds. Subregion: start '[92, 184, 276, 0]' stop '[184, 276, 325, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 3.253587 seconds. Prediction took 1.411066 seconds. Subregion: start '[184, 184, 92, 0]' stop '[236, 275, 184, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 0.801311 seconds. Prediction took 0.1241 seconds. Subregion: start '[92, 276, 92, 0]' stop '[184, 281, 184, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 0.888439 seconds. Prediction took 0.194061 seconds. Subregion: start '[92, 276, 184, 0]' stop '[184, 281, 276, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 0.574891 seconds. Prediction took 0.105007 seconds. Subregion: start '[92, 276, 276, 0]' stop '[184, 281, 325, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 2.959048 seconds. Prediction took 1.205887 seconds. Subregion: start '[184, 184, 184, 0]' stop '[236, 275, 271, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 2.086948 seconds. Prediction took 1.176914 seconds. Subregion: start '[184, 0, 0, 0]' stop '[239, 92, 92, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 3.267073 seconds. Prediction took 1.389748 seconds. Subregion: start '[184, 0, 92, 0]' stop '[239, 92, 184, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 2.535163 seconds. Prediction took 1.138342 seconds. Subregion: start '[184, 0, 184, 0]' stop '[239, 92, 276, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 2.134019 seconds. Prediction took 0.968845 seconds. Subregion: start '[184, 0, 276, 0]' stop '[239, 92, 325, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 2.421931 seconds. Prediction took 1.131886 seconds. Subregion: start '[184, 92, 0, 0]' stop '[239, 184, 92, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 3.041931 seconds. Prediction took 1.400453 seconds. Subregion: start '[184, 92, 92, 0]' stop '[239, 184, 184, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 2.861679 seconds. Prediction took 1.166864 seconds. Subregion: start '[184, 92, 184, 0]' stop '[239, 184, 276, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 2.195062 seconds. Prediction took 0.837896 seconds. Subregion: start '[184, 92, 276, 0]' stop '[239, 184, 325, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 2.199233 seconds. Prediction took 1.209191 seconds. Subregion: start '[184, 184, 0, 0]' stop '[239, 276, 92, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 2.861389 seconds. Prediction took 1.3699919999999999 seconds. Subregion: start '[184, 184, 92, 0]' stop '[239, 276, 184, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 2.61378 seconds. Prediction took 1.132135 seconds. Subregion: start '[184, 184, 184, 0]' stop '[239, 276, 276, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 2.083633 seconds. Prediction took 0.584694 seconds. Subregion: start '[184, 184, 276, 0]' stop '[239, 276, 325, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 0.745961 seconds. Prediction took 0.098366 seconds. Subregion: start '[184, 276, 0, 0]' stop '[239, 281, 92, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 0.378521 seconds. Prediction took 0.066011 seconds. Subregion: start '[184, 276, 92, 0]' stop '[239, 281, 184, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 0.413614 seconds. Prediction took 0.094332 seconds. Subregion: start '[184, 276, 184, 0]' stop '[239, 281, 276, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 0.266429 seconds. Prediction took 0.030908 seconds. Subregion: start '[184, 276, 276, 0]' stop '[239, 281, 325, 2]'
Compute time (in minutes): 3.0
done
/Users/pwalczysko/miniconda3/envs/ilastik-test2/lib/python3.7/multiprocessing/semaphore_tracker.py:144: UserWarning: semaphore_tracker: There appear to be 1 leaked semaphores to clean up at shutdown
  len(cache))
/Users/pwalczysko/miniconda3/envs/ilastik-test2/lib/python3.7/multiprocessing/semaphore_tracker.py:144: UserWarning: semaphore_tracker: There appear to be 1 leaked semaphores to clean up at shutdown
  len(cache))
/Users/pwalczysko/miniconda3/envs/ilastik-test2/lib/python3.7/multiprocessing/semaphore_tracker.py:144: UserWarning: semaphore_tracker: There appear to be 1 leaked semaphores to clean up at shutdown
  len(cache))
/Users/pwalczysko/miniconda3/envs/ilastik-test2/lib/python3.7/multiprocessing/semaphore_tracker.py:144: UserWarning: semaphore_tracker: There appear to be 1 leaked semaphores to clean up at shutdown
  len(cache))
(ilastik-test2) pwalczysko@ls31618~/Work/omero-guide-ilastik/scripts$ 
```

</details>

**Comment by jburel** [comment]
It is saved in the directory when the script is run

**Comment by pwalczysko** [comment]
Yes, it produced 2 png images.

They look reasonable, but I have no idea why they are two (if these are the 2 channels of the first plane of the probability map, then why do they display multiple colors each ?)

image_0
image_1

**Comment by jburel** [comment]
You have 2 images b/c 2 images are analysed. The dataset contains two images

**Comment by pwalczysko** [comment]
good to merge fmpov

**Pull request merged**